Wikipedia/Code/OnThisDayViewController.swift

```diff
@@ -22,7 +22,17 @@ class OnThisDayViewController: ColumnarCollectionViewController {
     
     var isDateVisibleInTitle: Bool {
         didSet {
-            guard isDateVisibleInTitle, let language = events.first?.language else {
+            
+            // Work-around for: https://phabricator.wikimedia.org/T169277
+            // Presently the event looks to its first article preview when you ask it for the language, so if the event has no previews, no lang!
+            let firstEventWithArticlePreviews = events.first(where: {
```

Comment:
This might also cause an issue with the header not being localized in the correct language. See `OnThisDayHeaderViewController.swift:19`